dali/python/nvidia/dali/_autograph/pyct/testing/codegen.py

```diff
@@ -167,7 +167,7 @@ def generate_Assign(self):
     # Generate right-hand side
     value_node = self.generate_expression()
     # Put it all together
-    node = gast.Assign(targets=[target_node], value=value_node)
+    node = gast.Assign(targets=[target_node], value=value_node, type_comment=None)
```

Comment:
Would it work with older gast?

Reply:
And without this change on an older gast we get
```
======================================================================
ERROR: test_replace_code_block (pyct.test_templates.TemplatesTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/klecki/work/salvador/dali/dali/test/python/autograph/pyct/test_templates.py", line 114, in test_replace_code_block
    gast.Assign(
  File "/home/klecki/.local/lib/python3.10/site-packages/gast/gast.py", line 26, in create_node
    raise TypeError(
TypeError: Assign constructor takes either 0 or 3 mandatory arguments

```

Reply:
😱